package.json

```diff
@@ -95,6 +95,8 @@
     "react-router": "^5.1.0",
     "react-router-dom": "^5.1.0",
     "react-scripts": "^3.1.2",
+    "react-stripe-checkout": "^2.6.3",
+    "react-toastify": "^5.4.1",
```

Comment:
Why do you use `react-toastify` package ?

Reply:
It's pretty simple,
it is used for adding notification...

Reply:
You should not use this lib, because @dimas0211 already implemented notifications

Reply:
> You should not use this lib, because @dimas0211 already implemented notifications

ok, i've done